### PR TITLE
 wasn't being passed in destroyAPI() which was causing an error in me…

### PR DESCRIPTION
**Pull request by shyal**
…teor 1.4.0.1

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  version: '1.1.0',
+  version: '1.1.1',
   summary: 'Stars rating control',
   git: 'https://github.com/barbatus/stars-rating',
   documentation: 'README.md'
 });
 
 Package.onUse(function(api) {
-  api.versionsFrom(['1.0.3.2', '1.2']);
+  api.versionsFrom(['1.0.3.2', '1.4.0.1']);
```
Comment (barbatus) [inline package.js:10]: why is that?

File: stars_rating.js [hunk 2]
```diff
@@ -94,8 +94,12 @@ function addjQuryApi($el) {
   });
 }
 
-function destroyApi() {
-  $el.unbind('reset');
+function destroyApi($el) {
+  if ($el) {
```
Comment (barbatus) [inline stars_rating.js:98]: I think it's better is to check if self.firstNode is not null in the destroyed.

File: package.js (hunk 1 above)
Comment (shyal) [inline package.js:10]: I was under the impression api.versionsFrom was the range of versions this package could effectively run on. Maybe I was mistaken.

File: stars_rating.js (hunk 2 above)
Comment (shyal) [inline stars_rating.js:98]: Sure. Do you want me to make that change or will you just DIY? Certainly fixed my bug, but I'm admittedly not familiar with this code.

File: package.js (hunk 1 above)
Comment (barbatus) [inline package.js:10]: I think it's not the range but versions it's compatible with. Core package versions are selected based on them.

File: stars_rating.js (hunk 2 above)
Comment (barbatus) [inline stars_rating.js:98]: Ok, I'll DIY.

**Comment by barbatus** [comment]
New version published with the fix. Please check out.

**Comment by shyal** [comment]
Fixed! Thanks a lot.

**Pull request closed without merging**